src/grelu/io/genome.py

```diff
@@ -6,8 +6,49 @@
 import os
 from typing import List, Optional, Union
 
-import genomepy
 import pandas as pd
+import pyfaidx
+
+
+class Genome:
```

Comment:
Why do we need to replace the genomepy Genome class entirely? I think it might be better to have this new class cover only the case where the user supplies a custom genome, instead of all cases.